src/Moderators.svelte

```diff
@@ -0,0 +1,118 @@
+<script>
+  export let moderatorsData
+</script>
+
+<div class="container my-8 p-5 bg-white rounded-lg">
+  <h2 class="text-center text-4xl mb-5 py-5 font-semibold">
+    Our Moderators
+  </h2>
+
+  <div
+    class="grid grid-cols-1 gap-5 sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-1 xl:grid-cols-2"
+  >
+
+  {#each  moderatorsData as data}
```

Comment:
This seems different usage to the existing card we have. 

I think the card should be reusable and only be a single card. The `each` loop should be outside the card. 

Similar to this...

```ts
{#each cardData as cardItem}
   <Card {...cardItem} />
{/each}
```

@BOLT04 what do you think?

Reply:
sure that works too I skipped the work of exporting variables .. if u want to add / subtract anything you can change the data

Reply:
I agree @eddiejaoude, we might want to use this card component for another context. So the component should only have from `<div class="bg-gray-0 h-32 md:h-48 lg:h-48">` forward. The HTML above that div goes to `App.svelte`.

I'd suggest to rename the file to something more generic, instead of `Moderators.svelte`. But it can be merged as is right now 😃.

Reply:
Cool is it merged yet ?

Reply:
it is not merged yet @UnevenCoder, I was under the impression you were still making changes.
Do you agree with the requested change in these comments? We can merge this as is and refactor in another PR if you prefer.